Feat/mobile table

RamComrax opened the pull request:
adjusting mobile layout of table according to xd.
did not apply logic to handle rotation of expand icon.

RamComrax committed:
adding custom dialog

including an example in components demo.
and 3 new icons to display within the dialogs.

RamComrax committed:
cleaning code and returning components application to its previous state

RamComrax committed:
adjusting table mobile layout according to XD

didn't apply the logic to handle the rotation of the expand icon.

israelComrax reviewed (commented):
why to cancel materiel arrow? call me for this

RamComrax committed:
adjusting mobile layout of table

applying rotation logic to custom expand icon.
switching px measurements to rem.
adjusting seperating lines between table rows.
removing duplicated display of rows in expand panel body.
adding פעולות headline to actions row.
adjusting layout of row actions icons according to XD.
adjusting table cell with button to have primary colored icon and a label with an underline.

RamComrax committed:
adjusting layout of table-cell

RamComrax committed:
retrieving commented actual Education project

RamComrax committed:
removing unused code
